tools/clang/test/CodeGenHLSL/SplatGroupSharedScalar.hlsl

```diff
@@ -0,0 +1,8 @@
+// RUN: %dxc -E main -T cs_6_0 %s
+
+groupshared int a;
+[numthreads(64, 1, 1)]
+void main() {
+  a = 123;
+  int4 x = (a).xxxx;
```

Comment:
This test should probably verify something... As-is we verify that it doesn't crash, but it doesn't actually verify correctness of any output.

Reply:
Done.